zulipterminal/server_url.py

```diff
@@ -0,0 +1,79 @@
+import urllib
```

Comment:
This commit doesn't seem to just *move* the code across, but also changes it... so doesn't this really replace the first commit? Doesn't this make the first commit meaningless? Which version are we including? Since this is the start of this code, we can just add it to the right file in the first place :)

As per my note elsewhere, if we bring code across from the server (which this looks like), we should indicate details, and what has changed or not - as well as anything else that you used in the process that's relevant.

Reply:
You are correct, the last commit ultimately makes the first commit useless. I have removed the first commit. :)

I have added brief comments for each new function added. If you want I can reword those comments to be more verbose.